spampd.pl

```diff
@@ -1006,7 +1007,7 @@ ($$$)
 				syslog_ident => 'spampd',
 				syslog_facility => 'mail',
 				background => $background,
-				# setsid => 1,
+				setsid => $background,
```

Comment:
I'd say also optional, perhaps enabled by default?  It was commented for a reason, though of course at this point I have memory of why.  I just tried to read up on setsid() a little... I understand the words (sounds good) but not all the implications by far.

Reply:
I would assume that this was commented out because it broke "--nodetach".

Reply:
Not sure... it first appears, already commented, in v2.10, before --nodetach was added, and after background was already implemented (in 2.0).